app/src/main/res/values/strings.xml

```diff
@@ -621,12 +621,12 @@
     <string name="overwrite_unrelated_warning">A file with this name already exists</string>
     <string name="overwrite_finished_warning">A downloaded file with this name already exists</string>
     <string name="overwrite_failed">cannot overwrite the file</string>
-    <string name="download_already_running">There is a download in progress with this name</string>
-    <string name="download_already_pending">There is a pending download with this name</string>
+    <string name="download_already_running">Downloading something with this name</string>
```

Comment:
```suggestion
    <string name="download_already_running">Downloading something with this name…</string>
```

Reply:
This has a different meaning, it seems like a download has just been started, while what the string should say is that a download with the same name is already in procress. What's wrong with the old string?

Reply:
Added "Already"